### DISCUSSION TITLE
How can I create multiple workspace?

### DISCUSSION THREAD

**Discussion by nelsonvu** [docmost/docmost#210] [General]
I just have started run this project on my local, but I can only create 1 workspace. Then, I. use the other browser but only show the login page.
Can you help me this issue?

**Comment by IvnSoft** [comment]
You have to 'invite' other people to join (through the settings). The invitation is done via email, so make sure to have SMTP working. I couldn't be bothered, so I just inserted new users directly into the PostgreSQL database XD

**Reply by smaciver** [reply]
Huh?? To create a new workspace, click name top right then choose workspace settings. Then left menu under Workspace click spaces. You should see a create workspace button. Surely OP has figured this out by now.

**Reply by Joonas12334** [reply]
Not space. A workspace. These are different things. Eg i want to manage two companies. I would need two different workspaces for those. Spaces are for different departments.. But I have the same questions regarding workspaces.

**Reply by smaciver** [reply]
oh sorry that I don't know. I assumed you meant space because it took me a minute to find where to create new spaces. It seems like it kind of just defaults into one main space per app and doesn't treat workspaces the same as others I guess but don't quote me on that.

**Comment by Philipinho** [comment]
Multi-workspace is a cloud-only feature which allows use to offer a cloud hosted version.
